api_examples/sequential_learning_api_tutorial.ipynb

```diff
@@ -0,0 +1,692 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# PyCC Sequential Learning Tutorial\n",
+    "\n",
+    "*Authors: Edward Kim, Enze Chen*\n",
+    "\n",
+    "In this notebook, we will cover how to perform **sequential learning** (SL) using the [Citrination API](http://citrineinformatics.github.io/python-citrination-client/). Sequential learning is the key workflow which allows machine learning algorithms and in-lab experiments to iteratively inform each other.\n",
```

Comment:
would be nice to link to [sequential learning on citrine.io](https://citrine.io/platform/sequential-learning/), or another resource that elaborates more on this